pyproject.toml

```diff
@@ -31,7 +31,7 @@ build-backend = "poetry.core.masonry.api"
 toml = ">=0.10.2"
 pytest = "^7.1.1"
 pytest-mock = "^3.7.0"
-cfn-lint = "^0.61.0"
+cfn-lint = "^0.65.1"
```

Comment:
fun fact: version 0.61 of cfn-lint still does not know 'AWS::CloudFront::OriginAccessControl'.

Reply:
And the 0.65?

Reply:
0.65 knows it